content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+## Overview
+Datadog Synthetics monitors your applications and API endpoints via simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing, as experienced by your users.
```

Comment:
- Datadog Synthetics monitor
- Synthetics helps you ensure uptime, identify regional issues, and make sure key business transactions can be performed on your application.

In my opinion, performance monitoring is more an APM & RUM thing